src/Octoshift/AwsArgs.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace OctoshiftCLI;
+
+public class AWSArgs
+{
+    public string AwsBucketName { get; set; }
+    public string AwsAccessKey { get; set; }
+    public string AwsSecretKey { get; set; }
+    public string AwsSessionToken { get; set; }
+    public string AwsCredentialType { get; set; }
```

Comment:
How does this ever get set? I can't see any sign of it in the code - but I'm probably missing something.

Reply:
I want to use an AWSArgs object instead of passing too many args.
Setting the value in [src/gei/Commands/MigrateRepoCommand.cs ](https://github.com/leonli-xero/gh-gei/blob/Add-AWS-credential-provider/src/gei/Commands/MigrateRepoCommand.cs#L182-L189)
Use the values in the file [src/Octoshift/AwsApi.cs](https://github.com/leonli-xero/gh-gei/blob/Add-AWS-credential-provider/src/Octoshift/AwsApi.cs#L81)